developer/src/ext/scfontcombobox/Makefile

```diff
@@ -1,18 +0,0 @@
-#
-# scFontCombo Makefile
-#
-
-!include ..\..\Defines.mak
-
-build: dirs
-    @-mkdir obj\Win32\$(TARGET_PATH)
-    $(DCC32DPK) scFontCombo.dpk
-    $(MAKE) install
```

Comment:
This part didn't appear to land in the corresponding build.sh.  Its `build` action appears to come from the `install` target below instead.  (Which I suppose covers the third line, but not the first two.)  Should it still be handled somewhere?

Reply:
DCC32DPK is the old way of building, replaced with msbuild. And it creates the output folder automatically. So `do_build` in `build.sh` does the same things as this.